base_delivery_carrier_label/models/stock_picking.py

```diff
@@ -85,7 +92,7 @@ def get_shipping_label_values(self, label):
             'datas_fname': label.get('filename', label['name']),
             'res_id': self.id,
             'res_model': 'stock.picking',
-            'datas': label['file'].encode('base64'),
+            'datas': b64encode(label['file']),
```

Comment:
I made a change in the base module that I do not know if it could cause problems for other integrations. But in this case, when we take the zpl label in txt format it is necessary to use b64encode instead of '' .encode ('base64') because this last method fails.
For example:
In a file what ends with:
^FN32^FD1/1^FS
^FN33^FD1.0^FS
^FN34^FD^FS
^FN35^FD^FS
^FN36^FD^FS
^XZ

With ''.encode('base64') the attachment ends with:
^FN32^FD1/1^FS
^FN33^FD1.0^FS
^FN34^FD^FS
^FN35^FD^FS
^FN36^FD^FS
^

I don't know enough of encoding to know the reason. But this change fixed the problem.